docs/docs/guides/contributing-components.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Contributing Components
+sidebar_label: Contributing Components
+---
+
+:::warning
+
+This functionality is under development. With these docs, we aim to elicit feedback from our community (you!) and focus our development efforts. Currently, we do not offer support for 3rd party components, but we are working toward supporting this.
+
+:::
+
+`Airy Core` is fundamentally a collection of components. When you [install `Airy Core`](../getting-started/installation/introduction.md), you install a set of _core_ components which create the minimum infrastructure to install other components and begin using the platform.
+
+These core components are:
+
+1. airy-controller
+2. api-admin
+3. api-communication
+4. api-websocket
+5. frontend-inbox
+6. frontend-control-center
+
+Airy's real value comes from the additional components you install through our marketplace in the Control Center UI of your Airy Instance. This documentation is on the processes we follow to create, update and store our components.
+
+## Components
+
+Airy is designed to support a plug-and-play architecture. Components are installed and run as pods in your cluster. Components can provide four kinds of functionality:
+
+1. Connector - i.e Rasa, DialogFlow
+2. Source - i.e., Facebook, WhatsApp, SMS
```

Comment:
We decided to not differentiate between `sources` and `connectors` on a component level. 

In the [glossary docs](https://docs.google.com/document/d/1MjLF1eylIgMIhxzP7kJHHroRQXeKQyO4-824MR0pplE/edit) I suggested the following definition for a connector which nicely brackets both use cases:
>  A connector is a type of component that authenticates with one or more third-party services in order to exchange data

This document should link to the glossary definitions that aren't here yet, unfortunately. @AitorAlgorta perhaps we can already add definitions for `component` and `connector` in this PR?